queries.sql

```diff
@@ -13,3 +13,29 @@ SELECT id, name, escape_attempts, weight_kg FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
+COMMIT;
+
```

Comment:
- [ Required ] I think it would be nice if you add a query to answer this question from the project requirements, please see the screenshot below. The question should be answered in it's on `Transaction`.

## Project requirements
![Screenshot from 2023-09-29 15-24-15](https://github.com/kazim110/Vet-Clinic/assets/16392046/1ca4e093-0d1a-4fa9-b6c7-90ef29920971)